book/introduction.md

```diff
@@ -0,0 +1,66 @@
+# Introduction
+
+<img src="./innersource-patterns-book-cover.jpg" title="InnerSource Patterns">
+
+{% hint style="info" %}
+This is an early release of the InnerSource Patterns book.
+You may still find broken links, spelling mistakes, or other errors in this book.
+Please help us to fix them to produce the best book possible :). Learn here how to [contribute to this book](../book/contribute-to-this-book.md).
+{% endhint %}
+
+Welcome to the **InnerSource Patterns book**.
+
+This book contains InnerSource best practices codified in a specific format to make it easy to understand, evaluate, and apply them in your context. We call this format a **pattern**.
+
+These patterns have been collected by the [InnerSource Commons](http://innersourcecommons.org) community over many years. The most mature of these patterns have been published in this book.
```

Comment:
```suggestion
These patterns have been collected by the [InnerSource Commons](http://innersourcecommons.org) community over many years. The most mature of patterns have been published in this book. Mature in this context means that each pattern has been reviewed by members of the community, and has at least one known instance where this pattern has been used.
```

I think pointing to the Contributor Handbook here to explain the levels and which ones are in the book would be too much for new readers.

Therefore I modified the description to explain briefly what we mean with _mature_ in this context.

Would that work?